files/build_templates/swss.service.j2

```diff
@@ -21,6 +21,8 @@ ExecStartPre=/usr/bin/docker exec database redis-cli -n 2 FLUSHDB
 ExecStartPre=/etc/init.d/sxdkernel start
 ExecStartPre=/usr/bin/mst start
 ExecStartPre=/sbin/modprobe i2c-dev
+# i2c_ismt driver resides on the same bus as lpc_i2c, we remove it since its not used in Mellanox box to allow lpc_i2c to be inserted
+ExecStartPre=-/sbin/rmmod i2c_ismt
```

Comment:
does this belong to swss? shouldn't this be included in your hw_management package?